src/test/scala/mesosphere/marathon/MarathonSchedulerActorTest.scala

```diff
@@ -158,10 +157,10 @@ class MarathonSchedulerActorTest extends AkkaUnitTest with ImplicitSender with G
       expectMsg(TasksReconciled)
 
       val nonTerminalTasks = instance.tasksMap.values.filter(!_.task.isTerminal)
-      assert(nonTerminalTasks.size == 7, "We should have 7 non-terminal tasks")
+      assert(nonTerminalTasks.size == 6, "We should have 7 non-terminal tasks")
```

Comment:
Why did you remove the `.addTaskCreated`? Wouldn't it be OK to leave it in and make sure it's not considered?

Reply:
➕

Reply:
because this test tests for nonterminal tasks (that they are not included). I added another test covering created.